### PR TITLE
Install Red Hat OpenShift Serverless and Red Hat Integration - Camel K Operators

### LINKED ISSUE #108: Smart Village: Install Red Hat OpenShift Serverless and Red Hat Integration - Camel K Operators
The [Smart Village Collaboratory project](https://research.redhat.com/blog/research_project/creating-a-global-open-research-platform-to-better-understand-social-sustainability-using-data-from-a-real-life-smart-village/) would like to install the `Red Hat OpenShift Serverless` and `Red Hat Integration - Camel K` operators to NERC. This would allow us to request scale up the number of pods running research simulations for smart traffic lights in parallel at runtime. Our students and professor at BU on the Smart Village project would like to add this kind of scalability to the research we are doing. The `Camel K` operator goes with the `OpenShift Serverless` operator to add more types of integrations, like Kafka to Serverless. The [Smart Village project has begun integrating with OpenShift Serverless here](https://github.com/computate-org/smartabyar-smartvillage/tree/main/openshift/kustomize/bundles/serverless/app).

### PR DESCRIPTION
**Pull request by computate**
The [Smart Village Collaboratory project](https://research.redhat.com/blog/research_project/creating-a-global-open-research-platform-to-better-understand-social-sustainability-using-data-from-a-real-life-smart-village/) would like to install the Red Hat OpenShift Serverless and Red Hat Integration - Camel K operators to NERC. This would allow us to request scale up the number of pods running research simulations for smart traffic lights in parallel at runtime. Our students and professor at BU on the Smart Village project would like to add this kind of scalability to the research we are doing. The Camel K operator goes with the OpenShift Serverless operator to add more types of integrations, like Kafka to Serverless. The [Smart Village project has begun integrating with OpenShift Serverless here](https://github.com/computate-org/smartabyar-smartvillage/tree/main/openshift/kustomize/bundles/serverless/app).

closes https://github.com/OCP-on-NERC/operations/issues/108

### CODE REVIEW

File: cluster-scope/base/operators.coreos.com/subscriptions/serverless-operator/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serverless-operator
+  namespace: openshift-serverless
```
Comment (naved001) [inline cluster-scope/base/operators.coreos.com/subscriptions/serverless-operator/subscription.yaml:5]: does this namespace already exist?

**Comment by larsks** [comment]
Same comment here as [this one](https://github.com/OCP-on-NERC/nerc-ocp-config/pull/234#issuecomment-1511477553) re: commit message formatting.

File: cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml [hunk 2]
```diff
@@ -10,7 +10,9 @@ resources:
 - ../../bundles/xdmod-reader
 - feature/odf
 - feature/custom-routes
-- ../../base/core/namespaces/openshift-gitops
```
Comment (larsks) [inline cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml]: Removing this namespace doesn't appear to be related to the rest of this PR. Was this intentional?

File: cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml [hunk 3]
```diff
@@ -10,7 +10,9 @@ resources:
...
+- ../../base/core/namespaces/openshift-serverless
+- ../../base/operators.coreos.com/subscriptions/red-hat-camel-k
+- ../../base/operators.coreos.com/subscriptions/serverless-operator
```
Comment (larsks) [inline cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml]: If you are installing an operator into a new namespace you will also need to create the appropriate OperatorGroup. Since you're creating three resources for the serverless operator, you should move it into a *bundle* in `cluster-scope/bundles` and then include that in the overlay `kustomization.yaml`.

**Review by larsks — CHANGES_REQUESTED** [review]

File: cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml (hunk 2 above)
Comment (computate) [reply cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml]: Thanks @larsks , I put it back in.

File: cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml (hunk 3 above)
Comment (computate) [reply cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml]: Sounds good @larsks , I added the operator group and the bundle.

**Review by jtriley — APPROVED** [review]

**Review by larsks — APPROVED** [review]
